### PR TITLE
DTSBPS-775: fix spring cve error

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
DTSBPS-775

### Change description ###
fix spring cve error

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/data.tf [hunk 1]
```diff
@@ -7,8 +7,8 @@ locals {
   aat_cft_vnet_name           = "cft-aat-vnet"
   aat_cft_vnet_resource_group = "cft-aat-network-rg"
 
-  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
-  app_aks_network_rg_name = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-network-rg" : "aks-infra-${local.aks_env}-rg"
+  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" || var.env == "preview" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
```
Comment (jasonpaige) [inline infrastructure/data.tf:10]: `app_aks_network_name    = var.env == "prod" ? "core-${local.aks_env}-vnet" : "cft-${local.aks_env}-vnet"`

File: infrastructure/data.tf [hunk 2]
```diff
@@ -7,8 +7,8 @@ locals {
...
+  app_aks_network_rg_name = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" || var.env == "preview" ? "cft-${local.aks_env}-network-rg" : "aks-infra-${local.aks_env}-rg"
```
Comment (jasonpaige) [inline infrastructure/data.tf:11]: app_aks_network_rg_name = var.env == "prod" ? "aks-infra-${local.aks_env}-rg" : "cft-${local.aks_env}-network-rg"

**Review by jasonpaige — APPROVED** [review]

File: infrastructure/data.tf (hunk 2 above)
Comment (rishikrsharma) [reply infrastructure/data.tf]: done

File: infrastructure/data.tf (hunk 1 above)
Comment (rishikrsharma) [reply infrastructure/data.tf]: done

**Pull request merged**